### DISCUSSION TITLE
Module '"zod"' has no exported member 'ZodTypeAny'. ts(2305)

### DISCUSSION THREAD

**Discussion by josh-i386g** [colinhacks/zod#3020] [Q&A] [answered]
```ts
import type { ZodTypeAny } from "zod";
```

Causes this TypeScript error:

```
Module '"zod"' has no exported member 'ZodTypeAny'. ts(2305)
```

Any idea why it's doing that? Weird as hell.

**Comment by josh-i386g** [comment]
Another weird thing is I cannot just import from `zod/lib/types`

![image](https://github.com/colinhacks/zod/assets/150557754/cb9c4b84-463f-4494-9069-19e9cb6047ca)

It requires me to reference the `node_modules` folder itself, like `../../node_modules/zod/lib/types`.

**Comment by JacobWeisenburger** [comment, marked as answer]
Please create a code sandbox or a repo so someone can look into it
